tests/istanbul/eip152_blake2/test_blake2.py

```diff
@@ -532,7 +532,7 @@ def test_blake2b_invalid_gas(
 
 @pytest.mark.valid_from("Istanbul")
 @pytest.mark.parametrize("call_opcode", [Op.CALL, Op.CALLCODE])
-@pytest.mark.parametrize("gas_limit", [100_000_000, 90_000, 110_000, 200_000])
+@pytest.mark.parametrize("gas_limit", [Environment().gas_limit, 90_000, 110_000, 200_000])
```

Comment:
@reedsa let me know what you think of this since you were the original author of the test.

The idea is that the bump occurs automatically when we bump the default in `CURRENT_MAINNET_BLOCK_GAS_LIMIT` or when we try to fill with a higher block gas limit passed as parameter.

Reply:
Good idea, I think these original values are pretty arbitrary. It's great having the ability to set this in a param!

Reply:
I wrote a unit test for this pattern and actually found a bug:
I was overwriting the `EnvironmentDefaults` in a fixture, but if the test uses `Environment().gas_limit` to generate its parametrization like in this example, pytest only resolves fixtures during actual test execution, so too late for this.
I modified the `EnvironmentDefaults` mod to happen during `pytest_config` which happens before collection even.